### ISSUE TITLE
remove behat from composer

### ISSUE THREAD

**Issue by austinpapp** [PowerlineApp/powerline-server#263]
we aren't using it.

**Comment by jterps08** [comment]
Twig would be needed for creating/modifying the superuser admin panel. Don't know about Behat.

**Comment by austinpapp** [comment]
@jterps08 yup. just reviewing as i see things here. this is spawned from me trying to sort out some build procedures and clean up the `build.xml` to be useful for us. i'd much rather travis do this than TC.
